flexy.properties

```diff
@@ -0,0 +1,3 @@
+ewon.address=10.0.0.53
```

Comment:
Good change. The changes made in this commit should be simultaneously carried over to https://github.com/hms-networks/sc-java-maven-starter-project to ensure we don't have too much divergence in our projects

Reply:
@alexjhawk 
It isn't perfect, but the idea is that a developer could use git update-index --assumed-unchanged flexy.properties. Then they could change this file to match their Flexy. I was getting tired of stash and stash pop during rebases and testing. 
git update-index --assume-unchanged can cause headaches when switching between branches when one branch doesn't have tracking turned off, so that is why this isn't perfect.

Reply:
At the very least, it provides a much better location for changing those three variables.

I haven't run into a scenario where I've had to do stash/pop at all with the pom.xml, but I can see where it could become annoying. (IntelliJ might be doing the stash/pop on its own when I edit a commit with the Git integration)